src/config/field-rules.js

```diff
@@ -1,3 +1,4 @@
+import optionSet from './disabled-on-edit/optionSet.js';
```

Comment:
I don't get this import; is it needed here?

Reply:
Hehe, good catch. Probably just something VSCode included when I typed `optionSet`...